src/pages/Marketplace/Cards/Card.tsx

```diff
@@ -60,7 +60,7 @@ export default function Card({
             </p>
             <p className="line-clamp-2">
               <span className="font-semibold">Active in:</span>{" "}
-              {isEmpty(active_in_countries)
+              {!active_in_countries || isEmpty(active_in_countries)
```

Comment:
@SovereignAndrey, should first update `type EndowmentBase` in `src/types/aws/ap/index.ts` so not to miss other fields

```ts
type EndowmentBase = {
  //...
  active_in_countries?: string[];
}
```